src/main/java/edu/kis/vh/nursery/list/ListInterface.java

```diff
@@ -0,0 +1,10 @@
+package edu.kis.vh.nursery.list;
+
+public interface ListInterface {
```

Comment:
Names of operations do not correspond to operating the list (or stack), thus do not reflect the interface name. 
Use proper method names. The names like countIn or peekaboo should remain only in rhymers.